tests/templates/test_subroutines/test_kupccgsd.py

```diff
@@ -624,3 +624,20 @@ def test_torch(self, tol):
         grads2 = [weights.grad]
 
         assert np.allclose(grads[0], grads2[0], atol=tol, rtol=0)
+
+
+class TestGradient:
+    """Test that the parameter-shift rule for this template matches that of backprop."""
+
+    def test_ps_rule_gradient(self, tol):
+        """Test parameter-shift rule gradient."""
+
+        dev = qml.device("default.qubit", wires=4)
+
+        backprop_grad = qml.grad(qml.QNode(circuit_template, dev, diff_method="backprop"))
+        ps_rule_grad = qml.grad(qml.QNode(circuit_template, dev, diff_method="parameter-shift"))
+
+        weights = np.array([0.55, 0.72, 0.6, 0.54, 0.42, 0.65], requires_grad=True).reshape(1, -1)
```

Comment:
```suggestion
        weights = qml.numpy.array([0.55, 0.72, 0.6, 0.54, 0.42, 0.65], requires_grad=True).reshape(1, -1)
```

Looks like the change wasn't pushed 🙂

Reply:
🥷